assets/airlines/fin.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icao": "fin",
+    "name": "Finnair",
+    "callsign": {
+        "name": "Finnair",
+        "length": 3
+    },
+    "fleets": {
+        "long": [
+            ["A333",  8],
+            ["A359",  8]
+        ],
+        "default": [
+            ["A319",  9],
+            ["A320", 10],
+            ["A321", 16],
+            ["AT72", 11],
+            ["E190", 12]
```

Comment:
@ij1 Quite true, I've never have really added other sections to airline files, but it does say in airline-format.md that "A default fleet must be defined, this should generally contain all the aircraft which the operator would fly."